beacon_node/execution_layer/src/lib.rs

```diff
@@ -158,72 +158,60 @@ impl ExecutionLayer {
         let Config {
             execution_endpoints: urls,
             builder_endpoints: builder_urls,
-            mut secret_files,
+            secret_files,
             suggested_fee_recipient,
             jwt_id,
             jwt_version,
             default_datadir,
         } = config;
 
-        if urls.is_empty() {
-            return Err(Error::NoEngines);
+        if urls.len() > 1 {
+            info!(log, "Only the first execution engine url will be used");
```

Comment:
```suggestion
            warn!(log, "Only the first execution engine url will be used");
```

Just a suggestion, might be nice to be louder about it.